lib/incoming_mail/handler.rb

```diff
@@ -72,7 +72,11 @@ def create_comment(msg)
       proposal = parsed_email.proposal
       user = parsed_email.comment_user
 
-      return unless user  # cannot create comment for non-existent user
+      # cannot create comment for non-existent user, or
+      # for user who is not already a subscriber
+      unless user && proposal.has_subscriber?(user)
```

Comment:
Maybe rename `create_comment(msg)` to `transform_message(msg)` and add a new method `create_method(user, proposal, text)` to handle the mechanics of actually creating the message? I can try that.